x/ccv/types/ccv.go

```diff
@@ -48,11 +50,6 @@ func (mat VSCMaturedPacketData) ValidateBasic() error {
 	return nil
 }
 
-func (mat VSCMaturedPacketData) GetBytes() []byte {
```

Comment:
This and the other GetBytes methods should never be used. VSCMatured and SlashPacket data are both wrapped as ConsumerPacketData when sent over the wire. The ConsumerPacketData.GetBytes() method should be used and is tested in this PR